package.json

```diff
@@ -13,63 +13,62 @@
-    "@aws-cdk/aws-redshift-alpha": "^2.18.0-alpha.0",
-    "@aws-cdk/aws-synthetics-alpha": "^2.18.0-alpha.0",
-    "@types/aws-lambda": "^8.10.110",
+    "@aws-cdk/aws-apigatewayv2-alpha": "^2.65.0-alpha.0",
+    "@aws-cdk/aws-redshift-alpha": "^2.65.0-alpha.0",
+    "@aws-cdk/aws-synthetics-alpha": "^2.65.0-alpha.0",
+    "@types/aws-lambda": "^8.10.114",
     "@types/http-errors": "^2.0.1",
-    "@types/jest": "^29.4.0",
-    "@types/lodash": "^4.14.191",
-    "@types/node": "^18.13.0",
+    "@types/jest": "^29.5.0",
+    "@types/lodash": "^4.14.192",
+    "@types/node": "^18.15.11",
     "@types/puppeteer": "^7.0.4",
     "@types/sax": "^1.2.4",
-    "@typescript-eslint/eslint-plugin": "^5.52.0",
-    "@typescript-eslint/parser": "^5.52.0",
-    "aws-cdk": "2.67.0",
-    "aws-sdk-client-mock": "^2.0.1",
-    "esbuild": "^0.17.8",
-    "eslint": "^8.34.0",
-    "eslint-config-prettier": "^8.6.0",
+    "@typescript-eslint/eslint-plugin": "^5.57.1",
+    "@typescript-eslint/parser": "^5.57.1",
+    "aws-cdk": "2.73.0",
+    "aws-sdk-client-mock": "^2.1.1",
+    "esbuild": "^0.17.15",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-config-standard": "^17.0.0",
-    "eslint-import-resolver-typescript": "^3.5.3",
+    "eslint-import-resolver-typescript": "^3.5.5",
     "eslint-plugin-ava": "^14.0.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-json": "^3.1.0",
-    "eslint-plugin-n": "^15.6.1",
+    "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-promise": "^6.1.1",
-    "jest": "^29.4.3",
-    "prettier": "^2.8.4",
-    "ts-jest": "^29.0.5",
+    "jest": "^29.5.0",
+    "prettier": "^2.8.7",
+    "ts-jest": "^29.1.0",
     "ts-node": "^10.9.1",
-    "typescript": "~4.9.5"
+    "typescript": "~5.0.3"
   },
   "dependencies": {
-    "@aws-lambda-powertools/logger": "^1.4.1",
-    "@aws-lambda-powertools/tracer": "^1.4.1",
-    "@aws-sdk/client-ec2": "^3.271.0",
-    "@aws-sdk/client-iam": "^3.271.0",
-    "@aws-sdk/client-s3": "^3.271.0",
-    "@aws-sdk/client-s3-control": "^3.271.0",
-    "@aws-sdk/client-secrets-manager": "^3.271.0",
-    "@aws-sdk/client-sfn": "^3.271.0",
-    "@aws-sdk/client-sqs": "^3.271.0",
-    "@aws-sdk/client-sts": "^3.271.0",
+    "@aws-lambda-powertools/logger": "^1.7.0",
+    "@aws-lambda-powertools/tracer": "^1.7.0",
+    "@aws-sdk/client-ec2": "^3.306.0",
+    "@aws-sdk/client-iam": "^3.306.0",
+    "@aws-sdk/client-s3": "^3.306.0",
+    "@aws-sdk/client-s3-control": "^3.306.0",
+    "@aws-sdk/client-secrets-manager": "^3.306.0",
+    "@aws-sdk/client-sfn": "^3.306.0",
+    "@aws-sdk/client-sqs": "^3.306.0",
+    "@aws-sdk/client-sts": "^3.306.0",
     "@middy/core": "^3.6.2",
-    "@middy/error-logger": "^4.2.7",
+    "@middy/error-logger": "^3.6.2",
     "@middy/http-json-body-parser": "^3.6.2",
-    "@middy/input-output-logger": "^4.2.7",
+    "@middy/input-output-logger": "^3.6.2",
```

Comment:
This looks suspect.  Appears to downgrade middy which we just for JSON object validation.

Reply:
its best to keep middy all the same version. and there were some issues with the latest middy release so we couldn't update them all with out risk of issue.

Reply:
after looking more closely I understand our current config to have mixed versions, and this PR downgrades add middy libraries to the same version
![image](https://user-images.githubusercontent.com/77642966/230634051-b4c4c99e-9096-4a3f-a438-6a216e89eca1.png)